src/app/shared/DisplayUtils.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * A function to call before passing a string to a markdown renderer.
+ * @param value the value to debrace
+ */
+export function debrace(value: string): string {
+  if (!value) {
+    return value;
+  }
+  let renderedValue = value.replace(/>/g, '&gt;').replace(/</g, '&lt;');
+
+  // if it replaced a <> in a markdown code block, change it back
+  renderedValue = renderedValue.replace(/`.+?`/g, match => {
```

Comment:
Initial thought: how does this handle something like this?
```
`banana` <script type="text/javascript">console.log("hi")</script> `apple`
```

Also triple backtick blocks?

Reply:
good call out - right now it's possible to get a script outside of a code block past the debrace step, but angular sanitizes it (by removing it from the content completely) in the next step before it's displayed, so it's not an XSS issue. still, that isn't intended behaviour - I'll see if it's possible to handle this on the tag renderer level instead.

triple code blocks just aren't handled 😛

Reply:
<img width="360" alt="Screenshot 2020-09-09 at 12 55 17" src="https://user-images.githubusercontent.com/8866981/92646746-bd6b4d00-f29b-11ea-822b-2890630178d4.png">

Reply:
Apparently enabling `sanitize: true` in the marked options makes it escape HTML correctly, but it's unfortunately a deprecated option 😐 

The recommended sanitize libraries simply remove any sanitized content from the DOM instead of escaping it, which isn't optimal. 

(update mid-writing this comment: `sanitize-html` has an escape rather than remove option, yay!)

https://github.com/markedjs/marked/issues/1232
https://marked.js.org/using_advanced#options
https://github.com/apostrophecms/sanitize-html

Reply:
upon further investigation, the deprecated `sanitize` option is the only option that works as expected. 😐